lib/manageiq/reporting/formatter/converter.rb

```diff
@@ -0,0 +1,23 @@
+module ManageIQ
+  module Reporting
+    module Formatter
+      class Converter
+        # generate a ruport table from an array of hashes where the keys are the column names
+        def self.hashes2table(hashes, options)
```

Comment:
:laughing: Did you show this 2 @Fryguy ?

Reply:
beautifully_named_methods4the_win